packages/frontend/pages/_app.tsx

```diff
@@ -22,6 +23,8 @@ type AppPropsWithLayout = AppProps & {
 const App = ({ Component, pageProps }: AppPropsWithLayout) => {
   const getLayout = Component.getLayout ?? ((page) => page);
 
+  locale.setLocale(i18n.language);
```

Comment:
I thin we need update locale value using user's date in other case backend will use default locale

Reply:
How we will change it if user has another locale?

Reply:
default locale will detect i18n

Reply:
ok